### PR TITLE
BTC support removal banner - Closes #4077

### LINKED ISSUE #4077: Inform users about the removal of BTC support
### Description
Since we'll drop the support of BTC tokens by v2.2.0, we should inform users about such change. @ann-vashchenko has prepared a design for a banner to accomplish said goal. 

### Motivation
Keep users informed about the changes.

### Acceptance criteria
- Implement according to the [design guide](https://www.figma.com/file/tkFg2nGUOvgn7vns9XwkGj/Banner%3A-BTC-Removal?node-id=0%3A1).
- Account for smaller screens.
- Do not display the banner if the user has already discarded it.

### PR DESCRIPTION
**Pull request by gadjacobs**
### What was the problem?

This PR resolves #4077 

### How was it solved?

-By adding adding a banner with a blog article to inform users about the discontinuation of bitcoin support on the wallet

### How was it tested?

Visually

### CODE REVIEW

File: src/components/toolbox/infoBanner/infoBanner.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { withTranslation } from 'react-i18next';
+// import { useSelector } from 'react-redux';
```
Comment (ikem-legend) [inline src/components/toolbox/infoBanner/infoBanner.js:3]: @gadjacobs Please remove the comments here

File: src/components/toolbox/infoBanner/infoBanner.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  infoLabel,
+  infoMessage,
+  infoLink,
+  className,
+  t,
+}) => {
+  const [visibility, setVisibility] = useState(
+    !localStorage.getItem(name) ? 'visible' : 'hidden',
+  );
+  // const isLoggedIn = useSelector(state => (state.account && state.account.passphrase));
```
Comment (ikem-legend) [inline src/components/toolbox/infoBanner/infoBanner.js:19]: @gadjacobs Also remove the comment here

File: src/components/toolbox/infoBanner/infoBanner.js (hunk 1 above)
Comment (gadjacobs) [reply src/components/toolbox/infoBanner/infoBanner.js:3]: Thanks. I commented it in testing to be able to test without having to login

**Review by ikem-legend — COMMENTED** [review]
@gadjacobs Looks good generally. All that's left is removing the comments

File: src/components/toolbox/infoBanner/infoBanner.js (hunk 1 above)
Comment (ikem-legend) [reply src/components/toolbox/infoBanner/infoBanner.js:3]: If so, then remember to update them before the PR gets merged

**Review by ikem-legend — APPROVED** [review]

**Review by ann-vashchenko — APPROVED** [review]

**Comment by ManuGowda** [comment]
verified the last commit LGTM

**Comment by reyraa** [comment]
There's a small issue in around `1045px`.

**Review by reyraa — APPROVED** [review]

**Pull request merged**